src/method.rs

```diff
@@ -440,6 +475,17 @@ mod test {
         assert_eq!(Method::GET, &Method::GET);
     }
 
+    #[test]
+    fn test_from_static() {
```

Comment:
Should probably also test that `assert_eq!(Method::from_static(b"GET"), Method::GET)`.